realm/migrate.gno

```diff
@@ -0,0 +1,85 @@
+package chess
+
+func checkMigrated() {
+	if migrated {
+		msg := "realm has migrated to " + string(migratedTo)
+		panic(msg)
+	}
+}
+
+func Migrate(newRealmPath string) string {
+	// TODO add check for path validity
```

Comment:
Leftover TODO?

Reply:
Not sure how to check for path validity at this point